### DISCUSSION TITLE
Is it possible to exploit this policy error?

### DISCUSSION THREAD

**Discussion by Balistic123** [3kh0/ext-remover#1200] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
i was looking through my school's policies and i found that the extensionallowlist has an error, is it possible to exploit this to allow me to use extension installer?    
![Screenshot 2024-10-22 9 33 00 AM](https://github.com/user-attachments/assets/66e887f2-b202-4d18-a98d-55cfa1529e74)

**Comment by HelloIloveGimkit2** [comment]
it's just your school being a dumbass

**Reply by HelloIloveGimkit2** [reply, marked as answer]
there is no exploit
